src/userdb/userdbctl.c

```diff
@@ -393,6 +408,8 @@ static int display_user(int argc, char *argv[], void *userdata) {
         else {
                 _cleanup_(userdb_iterator_freep) UserDBIterator *iterator = NULL;
 
+                match.fuzzy_names = argc > 1 ? argv + 1 : NULL;
```

Comment:
I'd like to make the arguments copied, for safety, especially to make it work even if we fork one day in a future.